code/controllers/subsystems/job.dm

```diff
@@ -752,6 +752,12 @@
 					if (leftovers)
 						leftovers += thing
 					Debug("EC/([H]): [thing] failed mask/suit/head check; leftovers=[!!leftovers]")
+				else if (G.slot == slot_back)
+					qdel(H.get_equipped_item(slot_back)) // Deletes starting backpack if you have a custom one
+					H.equip_to_slot_or_del(CI, G.slot)
+					to_chat(H, SPAN_NOTICE("Equipping you with [thing]!"))
```

Comment:
I don't think this should be here

Reply:
This chuck, from the if to the chat, does exactly the same thing as with any other loadout item, except it removes the default grey or department one before. If you could explain further?